compiler/ballerina-backend-jvm/src/main/ballerina/compiler_backend_jvm/jvm_method_gen.bal

```diff
@@ -886,8 +883,10 @@ function generateMainMethod(bir:Function userMainFunc, jvm:ClassWriter cw, bir:P
     mv.visitInsn(DUP);
     mv.visitInsn(ICONST_4);
     mv.visitMethodInsn(INVOKESPECIAL, SCHEDULER, "<init>", "(I)V", false);
+    mv.visitVarInsn(ASTORE, 1);
```

Comment:
Can you hardcode the var index to 1 here?

Reply:
Fixed